app/global-error.jsx

```diff
@@ -0,0 +1,23 @@
+'use client';
```

Comment:
In theory we can turn this file into a tsx. We just need to slap a `ts-ignore` above the `<Error />` component.